ftplugin/org.vim

```diff
@@ -152,7 +153,6 @@ fun CalendarAction(day, month, year, week, dir)
 	let msg = printf("Inserting %s | Modify date", g:org_timestamp)
 	exe s:py_version . "modifier = get_user_input('" . msg . "')"
 	" change date according to user input
-	exe s:py_version . "print modifier"
```

Comment:
To get the same behaviour in py2 you would need ``from __future__ import print_function``, and ``print modifier`` seemed unnecessary so I removed it.